test/e2e/test_tools.py

```diff
@@ -1,70 +1,148 @@
+        create_project(
+            input_file=self.test_project_minimal_file,
+            server=self.server,
+            user_mail=self.user,
+            password="test",
+            verbose=True,
+            dump=False
+        )
+
+        # open a "lists" section and the project that was created
+        with open("testdata/lists_multilingual_output_expected.json") as f:
+            lists_section = json.load(f)
+        with open(self.test_project_minimal_file) as f:
+            test_project_minimal = json.load(f)
+
+        # create a copy of the project that was created, and insert the first list into it
+        test_project_minimal_with_list_1 = copy.deepcopy(test_project_minimal)
+        test_project_minimal_with_list_1["project"]["lists"] = [lists_section[0], ]
+
+        # create another copy of the project that was created, insert the second list into it, and save it as file
+        test_project_minimal_with_list_2 = copy.deepcopy(test_project_minimal)
+        test_project_minimal_with_list_2["project"]["lists"] = [lists_section[1], ]
+        with open("testdata/tmp/test_project_minimal_with_list_2.json", "x") as f:
+            json.dump(test_project_minimal_with_list_2, f)
+
+        # The method to be tested can now be called with both versions of the same project. One is loaded from disk,
+        # the other is a Python object.They contain each another list.
+        name2iri_mapping1, success1 = create_lists(server=self.server,
+                                                   user=self.user,
+                                                   password=self.password,
+                                                   project_definition=test_project_minimal_with_list_1)
+        name2iri_mapping2, success2 = create_lists(server=self.server,
+                                                   user=self.user,
+                                                   password=self.password,
+                                                   input_file="testdata/tmp/test_project_minimal_with_list_2.json")
+        
+        # test that both lists have been correctly created
+        self.assertTrue(success1)
+        self.assertTrue(success2)
+        name2iri_names_1 = [str(m.path) for m in jsonpath_ng.ext.parse("$..* where id").find(name2iri_mapping1)]
+        name2iri_names_2 = [str(m.path) for m in jsonpath_ng.ext.parse("$..* where id").find(name2iri_mapping2)]
+        node_names_1 = [m.value for m in jsonpath_ng.ext.parse("$.project.lists[*]..name").find(test_project_minimal_with_list_1)]
+        node_names_2 = [m.value for m in jsonpath_ng.ext.parse("$.project.lists[*]..name").find(test_project_minimal_with_list_2)]
+        self.assertListEqual(name2iri_names_1, node_names_1)
+        self.assertListEqual(name2iri_names_2, node_names_2)
+
+
+    def test_validate_project(self) -> None:
+        self.assertTrue(validate_project(self.test_project_systematic_file))
```

Comment:
new

Reply:
Ideally, you would always test for the failure case too (even better, if you can test different failing scenarios).